### PR TITLE
169318195: Fix Teaching Sessions Widget bugs

### PR DESCRIPTION
**Pull request by yury-klika**
- Knowledge Owner is now displayed with their email, not id
- `New` status is a text now, not a table

### CODE REVIEW

File: packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemProgress.tsx [hunk 1]
```diff
@@ -1,36 +1,16 @@
...
-    return openApp(appId, { action: 'open-session', id: session['ogit/_id'] });
-  };
-
   if (!session['/teaching_step_progress']) {
-    return (
-      <Button secondary onClick={action}>
-        New
-      </Button>
-    );
+    return <>New</>;
```
Comment (alex-melnik) [inline packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemProgress.tsx]: Why not just `string` ?

File: packages/bootstrap/src/components/dashboard/types.ts [hunk 2]
```diff
@@ -97,9 +97,17 @@ export interface OgitSession extends OGIT.SafeNode {
   'ogit/Knowledge/archived': string;
 }
 
+export interface OgitAccount extends OGIT.SafeNode {
+  'ogit/name'?: string;
```
Comment (alex-melnik) [inline packages/bootstrap/src/components/dashboard/types.ts]: Is it optional field in `Account` vertex?

Comment (miraage) [reply packages/bootstrap/src/components/dashboard/types.ts]: It's not optional

File: packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemDetails.tsx [hunk 3]
```diff
@@ -1,29 +1,33 @@
...
-  'div b': {
-    marginRight: '5px',
-  },
-});
+export const SessionItemDetails: React.FC<SessionItemProps> = ({ session }) => {
+  const { client } = useWidgetContext();
+
+  const [{ loading, result }] = useLucene<OgitAccount>(client, {
+    querystring: session['/teaching_ownerId'] || '',
+  });
```
Comment (miraage) [inline packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemDetails.tsx]: No request should be sent upon empty ownerId

File: packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemProgress.tsx (hunk 1 above)
Comment (yury-klika) [reply packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemProgress.tsx]: Typescript prevents from returning just script, so I decided to return it this way, but you're right, it's nasty - I'll replace empty tag with `<span>`

Comment (miraage) [reply packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemProgress.tsx]: Oh, true. There is a very old issue: https://github.com/microsoft/TypeScript/issues/21699

Comment (miraage) [reply packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemProgress.tsx]: It should be shipped in Typescript 4.0 due to being a breaking change

Comment (yury-klika) [reply packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemProgress.tsx]: Replaced with `<span>New</span>`

File: packages/bootstrap/src/components/dashboard/types.ts (hunk 2 above)
Comment (yury-klika) [reply packages/bootstrap/src/components/dashboard/types.ts]: Removed at all - new approach does not use lucene to get data, and this interface becomes redundant

File: packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemDetails.tsx (hunk 3 above)
Comment (yury-klika) [reply packages/bootstrap/src/components/dashboard/widgets/SessionList/SessionItemDetails.tsx]: Replaced with `client.get()` inside of an `useEffect()` hook which made it possible to make the request conditional

**Pull request merged**